test/pathBuilder.test.js

```diff
@@ -71,15 +71,15 @@ describe('documentation paths', () => {
     });
 
     it('css distribution directory path should be correct', () => {
-        expect(pathBuilder.docsCssDistDir).toBe('./docs/dist/assets/css');
+        expect(pathBuilder.docsCssDistDir).toBe('./docs/dist/assets/dist/css');
```

Comment:
They worked before, the `docs/dist/assets/dist/` path was already in place, I didn't introduce it. 

I only added a new variable in this PR as there were issues with the path when the `assetSrcDir` is overridden.